lightly/transforms/utils.py

```diff
@@ -0,0 +1 @@
+IMAGENET_NORMALIZE = {"mean": [0.485, 0.456, 0.406], "std": [0.229, 0.224, 0.225]}
```

Comment:
Lets set in `collate.py` `imagenet_normalize = IMAGENET_NORMALIZE` so we are backwards compatible but it is only defined in one place. Please also add a comment in `collate.py` that `imagenet_normalize` is kept for backwards compatibility.